### DISCUSSION TITLE
No searches on cs.github.com return results

### DISCUSSION THREAD

**Discussion by elikoga** [community/community#32694] [Other Feature Feedback, Questions, & Ideas] [answered]
![grafik](https://user-images.githubusercontent.com/37669174/189552329-870c5301-26fd-4275-a890-2ecd2e8e1bcd.png)

Before:

![grafik](https://user-images.githubusercontent.com/37669174/189552339-fdb827fd-3fb5-4653-bb4b-f07cbf64a531.png)

**Comment by colinwm** [comment, marked as answer]
Hey @elikoga, we had an outage yesterday which has been resolved! Sorry about that, it should be working again.
